pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -1028,13 +1029,13 @@ func (r *multiCIDRRangeAllocator) createClusterCIDR(clusterCIDR *networkingv1alp
 	if updatedClusterCIDR.ResourceVersion == "" {
 		// Create is only used for creating default ClusterCIDR.
 		if _, err := r.client.NetworkingV1alpha1().ClusterCIDRs().Create(context.TODO(), updatedClusterCIDR, metav1.CreateOptions{}); err != nil {
-			klog.V(2).Infof("Error creating ClusterCIDR %s: %v", clusterCIDR.Name, err)
+			klog.V(2).InfoS("Error creating ClusterCIDR", "clusterCIDR", clusterCIDR.Name, "err", err)
```

Comment:
Is `clusterCIDR` another API object? Then use `klog.KObj(clusterCIDR)`, also below.

Reply:
done